code/game/objects/items/storage/smartpack.dm

```diff
@@ -176,7 +176,8 @@
 	update_icon(user)
 
 /obj/item/storage/backpack/marine/smartpack/proc/protective_form(mob/living/carbon/human/user)
-	if(!istype(user) || activated_form || immobile_form)
+	if(!istype(user) || activated_form || immobile_form || user.stat == DEAD)
```

Comment:
mob.is_dead() exists, not that u have to use it.

Reply:
There wouldn't be a difference between `mob.is_dead()` and `user.stat == DEAD` if we are talking about synths since synths dont fall unconscious.

If there is however, please let me know and thanks for your attention.